azimuth/types/model_performance.py

```diff
@@ -118,6 +118,8 @@ class ConfusionMatrixResponse(ModuleResponse):
     confusion_matrix: Array[float] = Field(..., title="Confusion Matrix normalized")
     class_names: List[str] = Field(..., title="Ordered class names for the confusion matrix")
     normalized: bool = Field(..., title="Normalized state of Confusion Matrix")
+    preserved_class_order: bool = Field(..., title="State of the order of the Confusion Matrix")
+    rejection_class_position: int = Field(..., title="Position of the rejection class")
```

Comment:
Did you see a problem with using "index" instead of "position"? It seems to be the vocabulary we are currently using in the API, with `pipeline_index`. To me, "position" could be confused with a one-based position, with the 1st position being the index 0.
```suggestion
    rejection_class_index: int = Field(..., title="Index of the rejection class")
```

Reply:
So the reason is that it's no longer the index of the rejection class, but its position in the matrix -> the indices are no longer ordered.

Reply:
I don't have an opinion about "position," but I think "index" overlaps with the actual index of the class (like, as specified by the user), which is different from its position in the confusion matrix.

Reply:
Oh, sorry, didn't see Gab's comment before I wrote mine! Exactly.

Reply:
Correct, but in the context of this response, the only "order" provided is the confusion matrix and the list of `class_names`. This number is the index in this `class_names` list. I think it would be correct.
That being said, if we worry about this, we can instead return the `rejection_class` as a string. I can do with that in the front end, it's fine.

Reply:
@JosephMarinier - wdyt? Do you want me to make the change?

Reply:
Ho sorry!
Yes, that's what I mean. I wouldn't really have to "search" for its position, as I am already looping over the matrix cells to render them, so it would simply mean changing the check to this:
```diff
-                columnIndex === data.rejectionClassPosition
+                data.classNames[columnIndex] === data.rejectionClass
```

Reply:
I can add the name of the rejection class to dataset info, it might be useful for other modules

Reply:
In addition, sure, if you wish, but I would like to have it here as I don't need `/dataset_info` in this page. It simplifies the front-end code if I don't need to handle loading and failure modes of `/dataset_info`. I think it goes very well with the `class_names` that you return.

Reply:
Ok, done!

Reply:
I replacing the line above in `webapp` but it's not working. Can you take a look?

Reply:
What was not working? It seems fine to me. I committed it.

Reply:
Yeah it was my mistake, I need to kill and restart the back end.